### PR TITLE
Readme content updates for 2.10

### PR DESCRIPTION
**Pull request by pderksen**

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -35,7 +35,8 @@ Do you prefer an eCommerce solution which is simple and intuitive or one which i
 Here are some of the features supported out-of-the-box:
 
 - **Frontend shopping cart** - Customers can browse your product catalogue, add items to their cart, and checkout when they are ready. Or they can bypass the cart using Buy Now buttons.
-- **Flexible and clean checkout form** - Buyers can complete their purchase with ease, directly on your site's checkout page.
+- **Highly customizable and clean checkout** - Buyers can complete their purchase with ease, directly on your site's checkout page.
+- **Payment flexibility** - Accept credit card payments using Stripe(https://stripe.com/) as well as PayPal payments. Apple Pay and Google Pay are also included.
```
Comment (ashleyfae) [inline readme.txt:39]:
```suggestion
- **Payment flexibility** - Accept credit card payments using [Stripe](https://stripe.com/) as well as PayPal payments. Apple Pay and Google Pay are also included.
```

File: readme.txt [hunk 2]
```diff
@@ -70,7 +72,7 @@ Payment gateways supported through free or premium extensions:
 * BitPay
 * Coinbase
 
-> What can I say? Stripe is my go-to payment processor and without this add-on I don't know what I would do. Connect EDD to your Stripe account in seconds and you're done! Having a simple test mode toggle is also a valuable development tool. This is another one of my go-to EDD extensions.
+> What can I say? Stripe is my go-to payment processor and without this integration I don't know what I would do. Connect EDD to your Stripe account in seconds and you're done! Having a simple test mode toggle is also super valuable.
```
Comment (ashleyfae) [inline readme.txt:75]: This is a direct quote, right? Should we be using square brackets to denote our changes?

**Review by ashleyfae — CHANGES_REQUESTED** [review]

File: readme.txt (hunk 1 above)
Comment (robincornett) [reply readme.txt:39]: With the ordering of the language here, it's not clear that Apple/Google Pay are included via Stripe.

Comment (cklosowski) [reply readme.txt:39]: `Apple Pay and Google Pay are also included via Stripe's Payment Request Buttons`

or something similiar

File: readme.txt (hunk 2 above)
Comment (cklosowski) [reply readme.txt:75]: If this is a direct quote from someone, we shouldn't modify it other than contextual hints we add, which should be in brackets.

File: readme.txt (hunk 1 above)
Comment (pderksen) [reply readme.txt:39]: @cklosowski Agreed with your edit.

File: readme.txt (hunk 2 above)
Comment (pderksen) [reply readme.txt:75]: Assuming it's a direct quote (but I'm not sure), sure let's put "integration" in square brakets.

The second sentence can be changed back to the original.

> Having a simple test mode toggle is also a valuable development tool.

And simply leave off the 3rd sentence.

File: readme.txt (hunk 1 above)
Comment (cklosowski) [reply readme.txt:39]: Ok @pderksen I'll get it all committed.

**Review by cklosowski — CHANGES_REQUESTED** [review]

File: readme.txt (hunk 2 above)
Comment (cklosowski) [reply readme.txt:75]: Ok, these are direct quotes from customers. I'll remove that 3rd sentence and put `integration` in square brackets.

**Review by cklosowski — APPROVED** [review]

File: readme.txt [hunk 3]
```diff
@@ -35,7 +37,8 @@ Do you prefer an eCommerce solution which is simple and intuitive or one which i
 Here are some of the features supported out-of-the-box:
 
 - **Frontend shopping cart** - Customers can browse your product catalogue, add items to their cart, and checkout when they are ready. Or they can bypass the cart using Buy Now buttons.
-- **Flexible and clean checkout form** - Buyers can complete their purchase with ease, directly on your site's checkout page.
+- **Highly customizable and clean checkout** - Buyers can complete their purchase with ease, directly on your site's checkout page.
+- **Payment flexibility** - Accept credit card payments using Stripe(https://stripe.com/) as well as PayPal payments. Apple Pay and Google Pay are also included via Stripe's Payment Request Buttons.
```
Comment (robincornett) [inline readme.txt:39]:
```suggestion
- **Payment flexibility** - Accept credit card payments using [Stripe](https://stripe.com/) as well as PayPal payments. Apple Pay and Google Pay are also included via Stripe's Payment Request Buttons.
```

File: readme.txt [hunk 4]
```diff
@@ -82,6 +84,7 @@ Easy Digital Downloads is a verified member of the [Stripe partner program](http
 
 With add-on plugins from our [extensions catalogue](https://easydigitaldownloads.com/downloads/?utm_source=extensions&utm_medium=description_tab&utm_content=extensions&utm_campaign=readme) you can super-charge your digital store. Some of our extension highlights include:
 
+* [Stripe Pro](https://easydigitaldownloads.com/downloads/stripe-gateway/?utm_source=extension&utm_medium=description_tab&utm_content=software-licensing&utm_campaign=readme) removes the additional 2% fee we charge as well as adds the ability to accept pre-approved payments (extension must be installed and activated to remove the additional 2% fee)
```
Comment (robincornett) [inline readme.txt:87]: @pderksen does this need to say "we charge" as it's not really us charging; it's an additional fee collected by Stripe? Also the parenthetical ends up feeling a bit redundant. Maybe simplify to this instead?

>Stripe Pro removes the additional 2% processing fee as well as adds the ability to accept pre-approved payments (extension must be installed and activated)

Comment (pderksen) [reply readme.txt:87]: @robincornett Yes much better wording. Approved!

Comment (robincornett) [inline readme.txt:87]:
```suggestion
* [Stripe Pro](https://easydigitaldownloads.com/downloads/stripe-gateway/?utm_source=extension&utm_medium=description_tab&utm_content=software-licensing&utm_campaign=readme) removes the additional 2% processing fee as well as adds the ability to accept pre-approved payments (extension must be installed and activated)

```

File: readme.txt [hunk 5]
```diff
@@ -82,6 +84,8 @@ Easy Digital Downloads is a verified member of the [Stripe partner program](http
 
 With add-on plugins from our [extensions catalogue](https://easydigitaldownloads.com/downloads/?utm_source=extensions&utm_medium=description_tab&utm_content=extensions&utm_campaign=readme) you can super-charge your digital store. Some of our extension highlights include:
 
+* [Stripe Pro](https://easydigitaldownloads.com/downloads/stripe-gateway/?utm_source=extension&utm_medium=description_tab&utm_content=software-licensing&utm_campaign=readme) removes the additional 2% processing fee as well as adds the ability to accept pre-approved payments (extension must be installed and activated)
```
Comment (robincornett) [inline readme.txt:87]: I think we want to double check the UTM tags in here please. Is that a @pderksen question?

Comment (pderksen) [reply readme.txt:87]: @robincornett Good catch. If you could please update to `&utm_content=stripe-pro`

Comment (robincornett) [reply readme.txt:87]:
```suggestion
* [Stripe Pro](https://easydigitaldownloads.com/downloads/stripe-gateway/?utm_source=extension&utm_medium=description_tab&utm_content=stripe-pro&utm_campaign=readme) removes the additional 2% processing fee as well as adds the ability to accept pre-approved payments (extension must be installed and activated)
```

**Review by robincornett — APPROVED** [review]

**Pull request merged**